scripts/migrations/05-migrate-agents-3tier.py

```
@@ -0,0 +1,346 @@
+  agent → section_title → directive content
+"""
+from __future__ import annotations
+
+import re
+import sys
+from pathlib import Path
+
+import yaml
+
+REPO_ROOT = Path("/Users/alexey.zhuchkov/Documents/Agents")
+REGISTRY = yaml.safe_load(Path("/tmp/registry_snapshot.yaml").read_text())
```

Ruff (0.15.13)

[error] Probable insecure usage of temporary file or directory: "/tmp/registry_snapshot.yaml"

(S108)